src/konami-code.js

```diff
@@ -41,7 +41,7 @@
     if (typeof module === "object" && module.exports) {
         module.exports = factory;
     }
-}(this, function callee(options) {
+}(this ?? globalThis, function callee(options) {
```

Comment:
Actually, I don't know that `konami-code-js` doesn't still explicitly support very old runtimes, so:

```suggestion
}(this || globalThis, function callee(options) {
```